### ISSUE TITLE
Doi lookup for new article not working

### ISSUE THREAD

**Issue by lc9275** [JabRef/jabref#4164]
JabRef 5.0-dev--snapshot--2018-07-02--master--aed7712bc
Windows 10 10.0 amd64 
Java 1.8.0_171

I am using the latest jabref snapshot and I was surprised to see that the "new article" dialog window is no longer there. I used this all the time to generated a new entry from a doi. Now it seems you have to go to the DOI field and generate the entry from there. However this is not working, ie nothing happens when I press either the  "lookup doi" or "generate bibtex from doi" button.

Steps to reproduce:

1. Click the "+" icon, new article entry
2. Go to the general tab and fill in a doi,  10.1103/PhysRev.47.777 
3. Press the  "lookup doi" or "generate bibtex from doi"  button
4. Nothing happens

**Comment by stefan-kolb** [comment]
The old dialog should be reachable via `Library -> new entry` or `ctrl-n`.

- [ ] However, there seems to be a problem @tobiasdiez @Siedlerchr 
The dialog appears in the background and let's JabRef "freeze" in the main window marking it as non responding.
- [ ]  We should readd a button for the old dialog!

Just for clarification:
- `lookup DOI` tries to find the DOI for an entry based on the BibTeX content
- `Generate BibTeX from DOI` fetches the BibTeX data based on the provided DOI

**Comment by marisuki** [comment]
Hi, I'm a freshman to OSS, mind if I make some enhancement? I together with my partner will work to fix it. :) And could you give me some help please such as the idea of which class or package would relate to his the problem? Thanks. :)

**Comment by tobiasdiez** [comment]
@marisuki Thanks for you interest. Sadly, I read your message only after fixing this issue myself in #4179. But you are invited to have a look at issues tagged with [good first issue](https://github.com/JabRef/jabref/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22), which should provide a good starting point.

**Comment by tobiasdiez** [comment]
Thanks for the reminder. There is also a fix in the pipeline for the issue concerning the "generate bibtex from doi" button and I'll will create a new issue for the button in the toolbar.

**Comment by marisuki** [comment]
Uh, I think I will try to solve that problem~"generate bibtex from doi".

**Comment by tobiasdiez** [comment]
This should be fixed in the latest development version. Could you please check the build from http://builds.jabref.org/master/. Thanks! Please remember to make a backup of your library before trying-out this version.
